core/src/test/java/org/hibernate/ogm/test/datastore/DatastoreProviderGeneratingSchema.java

```diff
@@ -51,32 +45,17 @@
 
 	public static class TestSchemaDefiner extends BaseSchemaDefiner {
 
+		public static boolean schemaInitialized = false;
+
 		@Override
 		public void initializeSchema(SchemaDefinitionContext context) {
-			for ( Namespace namespace : context.getDatabase().getNamespaces() ) {
-				for ( Table table : namespace.getTables() ) {
-					if ( table.isPhysicalTable() ) {
-						String tableName = table.getQuotedName();
-						// do something with table
-						Iterator<Column> columns = table.getColumnIterator();
-						while ( columns.hasNext() ) {
-							Column column = columns.next();
-							String columnName = column.getCanonicalName();
-							// do something with column
-						}
-						//TODO handle unique constraints?
-					}
-				}
-			}
-
-			throw new RuntimeException( "STARTED!" );
+			schemaInitialized = true;
```

Comment:
Looks like a good simplification but what was the issue exactly?

Reply:
The issue is that now in case of error during the factory creation no exception is thrown.
This is per J{PA specification.

This also caused some discussion on https://hibernate.atlassian.net/browse/HHH-12247